godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/GodotGooglePlayBilling.java

```diff
@@ -74,9 +73,12 @@ public class GodotGooglePlayBilling extends GodotPlugin implements PurchasesUpda
 	public GodotGooglePlayBilling(Godot godot) {
 		super(godot);
 
+		PendingPurchasesParams pendingPurchasesParams = 
+			PendingPurchasesParams.newBuilder().enableOneTimeProducts().build();
+
 		billingClient = BillingClient
 								.newBuilder(getActivity())
-								.enablePendingPurchases()
+								.enablePendingPurchases(pendingPurchasesParams)
```

Comment:
`enablePendingPurchases` is [deprecated](https://developer.android.com/reference/com/android/billingclient/api/BillingClient.Builder#enablePendingPurchases()): `This method is equivalent to enablePendingPurchases(PendingPurchasesParams.newBuilder().enableOneTimeProducts().build())`